Documentation/ApiOverview/Configuration/ConfigurationOverview.rst

```diff
@@ -43,10 +43,10 @@ Extension files
 ---------------
 
 :ref:`composer.json <composer-json>`
-   Composer configuration
+   Composer configuration, required in composer-based installations
```

Comment:
```suggestion
   Composer configuration, required in Composer installations
```

Reply:
I would keep Composer-based installation as a composer installation might be mistaken for the installation of composer (not with composer)